Kickstarter-iOS/Features/PledgedProjectsOverview/PPOContainerViewController.swift

```diff
@@ -20,6 +23,26 @@ public class PPOContainerViewController: PagedContainerViewController<PPOContain
       (.projectAlerts(.count(5)), ppoViewController),
       (.activityFeed(.dot), activitiesViewController)
     ])
+
+    let tabBarController = self.tabBarController as? RootTabBarViewController
+
+    self.ppoViewModel.$results.sink { [weak self] results in
+      let badge: TabBarBadge = results.values.count > 0 ? .count(results.values.count) : .none
+      self?.setPagedViewControllers([
+        (.projectAlerts(badge), ppoViewController),
+        (.activityFeed(.dot), activitiesViewController)
+      ])
+    }.store(in: &self.subscriptions)
+
+    ppoView.viewModel.navigationEvents.sink { nav in
+      switch nav {
+      case .backingPage:
+        tabBarController?.switchToProfile()
```

Comment:
Can we rename this case if it's supposed to refer to switching to the profile page? There's a separate thing called backing details or backing details page, which refers to the backing details within the survey webview. To me, `backingPage` sounds like it should be opening that, not the profile.